lib/sqlalchemy/dialects/mysql/base.py

```diff
@@ -1063,6 +1063,7 @@ class MyClass(Base):
 from ...sql import sqltypes
 from ...sql import util as sql_util
 from ...sql.compiler import InsertmanyvaluesSentinelOpts
+from ...sql.schema import SchemaConst
```

Comment:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4869